addons/scopes/stringtable.xml

```diff
@@ -4,78 +4,97 @@
             <Japanese>ACE スコープ調節を有効化</Japanese>
+            <Polish>Wymus uzycie poręteł regulacyjnych</Polish>
         </Key>
         <Key ID="STR_ACE_Scopes_forceUseOfAdjustmentTurrets_description">
             <English>Force usage of adjustmet turrets on high powered scopes</English>
             <Japanese>高倍率スコープで調整の使用を強制させます</Japanese>
+            <Polish>Wymuś użycie pokręteł regulacyjnych dla celowników o duzym powiększeniu</Polish>
         </Key>
         <Key ID="STR_ACE_Scopes_correctZeroing_displayName">
             <English>Correct zeroing</English>
             <Japanese>ゼロイン調節</Japanese>
+            <Polish>Poprawka zerowania </Polish>
         </Key>
         <Key ID="STR_ACE_Scopes_correctZeroing_description">
             <English>Corrects the zeroing of all small arms sights</English>
             <Japanese>全ての小口径用照準器のゼロインを調節します</Japanese>
+            <Polish>Poprawia zerowanie wszystkich celowników broni ręcznej</Polish>
         </Key>
         <Key ID="STR_ACE_Scopes_overwriteZeroRange_displayName">
             <English>Overwrite zero distance</English>
             <Japanese>ゼロイン距離を上書き</Japanese>
+            <Polish>Nadpisuje ustawienie dla zerowego dystansu </Polish>
         </Key>
         <Key ID="STR_ACE_Scopes_overwriteZeroRange_Description">
             <English>Uses the 'defaultZeroRange' setting to overwrite the zero range of high power scopes</English>
             <Japanese>'defaultZeroRange'設定を使う高倍率スコープのゼロイン距離を上書きします</Japanese>
+            <Polish>Używa/nadpisuje 'standardowej odległości zerowania' zamiast ustawionej odległości zerowania dla celowników o duzym przybliżeniu </Polish>
         </Key>
         <Key ID="STR_ACE_Scopes_defaultZeroRange_displayName">
             <English>Default zero distance</English>
             <Japanese>標準のゼロイン距離</Japanese>
+            <Polish>Domyślny dystans zerowy</Polish>
         </Key>
         <Key ID="STR_ACE_Scopes_defaultZeroRange_description">
             <English>High power scopes will be zeroed at this distance</English>
             <Japanese>高倍率スコープのゼロイン距離はこの設定になります</Japanese>
+            <Polish>Celowniki o duzym powiększeniu będą zerowane dla tej odległości</Polish>
         </Key>
         <Key ID="STR_ACE_Scopes_zeroReferenceTemperature_displayName">
             <English>Reference temperature</English>
             <Japanese>温度の参照</Japanese>
+            <Polish>Wzorcowa temperatura</Polish>
         </Key>
         <Key ID="STR_ACE_Scopes_zeroReferenceTemperature_description">
             <English>Temperature at which the scope was zeroed</English>
             <Japanese>スコープがゼロインされる温度</Japanese>
+            <Polish>Temperatura przy której celownik został wyzerowany</Polish>
         </Key>
         <Key ID="STR_ACE_Scopes_zeroReferenceBarometricPressure_displayName">
             <English>Reference barometric pressure</English>
             <Japanese>気圧の参照</Japanese>
+            <Polish>Standardowe cisnienie barometryczne</Polish>
```

Comment:
Yes, and here I had a problem, because there's much differences and severalways of finding (and setting) the attitude based on barometric (atmospheric) pressure... For instance setting the Raference barometric pressure at the airport looks a bit different than it can be in normal "field" use . So that I wasn't sure exactly what ACE programists mean by using "reference barometric pressure" phraze